docs/content/uijson_format/forms.rst

```diff
@@ -100,6 +100,36 @@ For a simple string parameter, use an empty ``str`` value to have an empty strin
     :height: 100
 
 
+Radio Label Parameter
+----------------------
+
+If both ``originalLabel`` and ``alternateLabel`` members are added to a string parameters, the dialog will render a
+two-choice radio button.  The ``value`` will reflect the current choice and all containing one of the selections will be
+updated with the choide of the radio button.
```

Comment:
I don't really get it. All forms after this one containing the strings in `value` get replaced?

Just read the docstring for the class - seems clearer. Maybe replicate here?

Reply:
re-worded